modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/di/RepositoryModule.kt

```diff
@@ -151,4 +153,7 @@ abstract class RepositoryModule {
     @Binds
     @Singleton
     abstract fun provideRatingsManager(ratingsManagerImpl: RatingsManagerImpl): RatingsManager
+
+    @Binds
+    abstract fun bindChapterManager(chapterManagerImpl: ChapterManagerImpl): ChapterManager
```

Comment:
```suggestion
    abstract fun provideChapterManager(chapterManagerImpl: ChapterManagerImpl): ChapterManager
```

I personally prefer `bind` instead of `provide`, but since all methods of this class use `provide` what do you think to change it?

Reply:
I don't care either way. I used bind because it actually binds but sounds good.